src/aqua/installation-spell/src/aqua/cli.aqua

```diff
@@ -2,6 +2,7 @@ import "./config.aqua"
 import "./deploy.aqua"
 import "./console.aqua"
 import "./deal_spell.aqua"
+import "./upload.aqua"
```

Comment:
Why?

Reply:
Can you please use `use X from ./upload.aqua` so it is obvious which functions are imported?